x-pack/test/api_integration/apis/endpoint/metadata.ts

```diff
@@ -161,7 +161,7 @@ export default function({ getService }: FtrProviderContext) {
           .post('/api/endpoint/metadata')
           .set('kbn-xsrf', 'xxx')
           .send({
-            filter: `host.os.variant.keyword:${variantValue}`,
+            filter: `host.os.variant:${variantValue}`,
```

Comment:
Since we're using ecs to generate the metadata template, `keyword` is no longer needed (it was added by the dynamic mapping functionality within ES)